tsconfig.json

```diff
@@ -5,6 +5,8 @@
     "incremental": false,
     // Required to run benchmark command from root directory
     "typeRoots": ["node_modules/@types", "./types"],
-    "noEmit": true
+    "noEmit": true,
+    // To be used in the test fixtures
+    "resolveJsonModule": true
```

Comment:
This is required to change here, because `yarn lint` on the repo is run with the top level config file, not the package specific.